bokehjs/src/less/attribution.less

```diff
@@ -3,8 +3,12 @@
   flex-direction: column;
   flex-wrap: nowrap;
   padding: 2px;
+  // @mattpap: So I think the 0.5 alpha value is to help this look good no
+  // matter what background color is used for the data viz, but this creates
+  // contrast issues when the background is dark, and text color of the
+  // attribution is also dark. Not sure what to do about this. Any thoughts?
```

Comment:
Alpha is there to make it see through, so that the annotation doesn't obscure the underlying content too much. I'm not sure what's the best practice to improve contrast issues.